### PR TITLE
v4.1.x: NEWS and VERSION updates for v4.1.3

### PR DESCRIPTION
**Pull request by jsquyres**
v4.1.3 checklist: #9703

bot:notacherrypick

### CODE REVIEW

**Comment by jsquyres** [comment]
@bwbarrett This is created in draft mode because I've asked @edgargabriel for some feedback on the OMPI NEWS bullets.

File: NEWS [hunk 1]
```diff
@@ -58,8 +58,34 @@ included in the vX.Y.Z section and be denoted as:
...
+  fixed MPI_REDUCE_SCATTER_BLOCK for similar issues with large payload
+  collectives.
+- Back-port ROMIO 3.3 fix to use stat64() instead of stat() on GPFS.
+- Fixed several non-blocking MPI collectives to not round fractions
+  based on float precision.
+- Fix compile failure for --enable-heterogeneous.  Also updated the
+  README to clarify that --enable-heterogeneous is functional, but
+  still not recomended for most environments.
+- Minor fixes to OMPIO, including:
+  - Fixing the open behavior of shared memory shared file pointers.
```
Comment (edgargabriel) [inline NEWS:81]: - Fixing the open behavior of shared file pointers

Comment (edgargabriel) [reply NEWS:81]: actually, leave it as it is, it is ok.

**Review by edgargabriel — APPROVED** [review]

File: NEWS (hunk 1 above)
Comment (jsquyres) [reply NEWS:81]: @edgargabriel and I conversed in Slack: we updated the text and I re-pushed.

**Review by bwbarrett — APPROVED** [review]

**Pull request merged**